packages/main/src/TabContainer.ts

```diff
@@ -438,18 +456,20 @@ class TabContainer extends UI5Element {
 
 		this._setItemsForStrip();
 
-		if (!this.shadowRoot.contains(document.activeElement)) {
+		if (!this.shadowRoot!.contains(document.activeElement)) {
 			const focusStart = this._getRootTab(this._selectedTab);
-			this._itemNavigation.setCurrentItem(focusStart);
+			this._itemNavigation.setCurrentItem(focusStart as ITabbable);
```

Comment:
You can try adding "implements ITabbable" to the Tab and perhaps the cast would be needed.

Reply:
Done